src/components/LicenseIcons.vue

```diff
@@ -68,7 +68,7 @@ export default LicenseIcons;
 
   .photo-license-icon {
     height: inherit;
-    margin-right: .5rem;
+    margin-right: .1rem;
```

Comment:
According to [the designs](https://www.figma.com/file/LR3KYeaqVkDYmgNm1rnsHx/Mockup---Sprint-1?node-id=0%3A335), I think a margin of `.3rem` is more accurate

Reply:
@brenoferreira Done :)